antora-ui-camel/src/css/header.css

```diff
@@ -425,6 +425,15 @@ body {
   }
 }
 
+#search-cancel {
+  position: relative;
+  bottom: 39.9%;
+  left: 90%;
```

Comment:
Could we use `rem` in these? Perhaps `padding-left` and `padding-top` instead (haven't tried it).